components/Template.js

```diff
@@ -67,7 +67,13 @@ function transformData(fn, templateKey, originalData) {
     data = fn(originalData);
   } else if (typeof fn === 'object') {
     // ex: transformData: {hit, empty}
-    data = fn[templateKey] && fn[templateKey](originalData);
```

Comment:
Yes it was like that before but I prefer to split it and add an explicit comment.